lib/dkdeploy/php/helpers/http.rb

```diff
@@ -41,7 +41,7 @@ def http_get_with_redirect(url, limit = 5)
           http.read_timeout = fetch :http_read_timeout
           if url.scheme == 'https'
             http.use_ssl = true
-            http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+            http.verify_mode = OpenSSL::SSL::VERIFY_NONE if disable_ssl_check
```

Comment:
@websi I guess you would like that configurable in the server properties and how should it be called?